R/supervised-model-development.R

```diff
@@ -298,10 +298,11 @@ SupervisedModelDevelopment <- R6Class("SupervisedModelDevelopment",
       self$params$df <- self$params$df[,colSums(is.na(self$params$df)) < nrow(self$params$df)]
 
       # Remove date columns
-      dateList <- grep("DTS$", colnames(self$params$df))
-      if (length(dateList) > 0) {
-        self$params$df <- self$params$df[, -dateList]
-      }
+      # dateList <- grep("DTS$", colnames(self$params$df))
+      # if (length(dateList) > 0) {
+      #   self$params$df <- self$params$df[, -dateList]
+      # }
```

Comment:
These comments should be deleted.

Reply:
Sorry forgot about that. I left it for reference. These are deleted.